src/test/kotlin/org/jitsi/videobridge/JitsiConfigTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge
+
+import io.kotlintest.Spec
+import io.kotlintest.extensions.TopLevelTest
+import org.jitsi.config.ConfigTest
+import org.jitsi.config.JitsiConfigFactory
+import org.jitsi.utils.config.ConfigSource
+import org.jitsi.videobridge.testutils.ConfigSourceWrapper
+
+/**
+ * This class is/should be a duplicate of the one in jicoco-kotlin.  We can't
+ * move it to jicoco-test-kotlin because it would cause a circular maven
+ * dependency
+ */
+abstract class JitsiConfigTest : ConfigTest() {
```

Comment:
We already have this class in jicoco: https://github.com/jitsi/jicoco/pull/63/files#diff-c83b2ca7ca45151e7b86062f3434064f

Reply:
Yes, but since it's a test class I didn't want to define it there (and I couldn't put it in `jicoco-test-kotlin`).  I guess I can put it in the main jicoco-kotlin code, even though it's for testing.  Is that what you think would be best?

Reply:
If it doesn't bring in a lot of other classes, I think moving to jicoco-kotlin would be better than having two copies.

Reply:
Hmm, unfortunately it'd drag the kotlintest framework along with it.

Reply:
Honestly the static singleton (`JitsiConfig`) really sucks.  It makes testing a pain and I'd really love to get rid of it (which would also likely eliminate the need for this sort of thing).